src/libstd/sys/windows/c.rs

```diff
@@ -732,7 +746,7 @@ if #[cfg(target_vendor = "uwp")] {
                                             lpFileInformation: LPVOID,
                                             dwBufferSize: DWORD) -> BOOL;
         pub fn BCryptGenRandom(hAlgorithm: LPVOID, pBuffer: *mut u8,
-                               cbBuffer: ULONG, dwFlags: ULONG) -> LONG;
+                                cbBuffer: ULONG, dwFlags: ULONG) -> LONG;
```

Comment:
Not sure what happened here with whitespace but tidy didn't complain. I can fix on a later commit.